### ISSUE TITLE
Localization function no longer works

### ISSUE THREAD

**Issue by shibuya-r** [unlayer/react-email-editor#425]
Hi,
I used to use it in Japanese, but the localization function no longer works.
Is this due to the new pricing plan?
If I don't have access to my old workspace, will I still be able to localize for free?

**Comment by omerzahidbajwa** [comment]
Yes, unfortunately, if you don't have an old workspace with an old project new billing plans will apply.

you can check them out at https://unlayer.com/pricing

**Comment by ildar979** [comment]
Can I manually add my localization in the free version?
